origami.json

```diff
@@ -22,25 +22,25 @@
 		"sass": "demos/src/demo.scss",
 		"js": "demos/src/demo.js",
 		"bodyClasses": "o-hoverable-on",
-		"dependencies": ["o-fonts@^1.5.0", "o-buttons@^4.0.0"]
+		"dependencies": ["o-fonts@^3.0.0", "o-buttons@^4.0.0"]
```

Comment:
This could cause conflicts in the demos at first, but I'm working on o-buttons now so it could be worth waiting until that's released to update this?